### PR TITLE
Makes pickaxe upgrades less horrible and irrelevant.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
"Pickaxes are obselete" -Kor Phaeron

Fuck you dad.

Upgrades efficiency and force of some pickaxes, making them less garbo, but not necessarily on par with the KA/
:cl:
tweak: Pickaxe upgrades are now better.
/:cl:

### CODE REVIEW

File: code/modules/mining/mine_items.dm [hunk 1]
```diff
@@ -81,18 +81,18 @@
 	name = "silver-plated pickaxe"
 	icon_state = "spickaxe"
 	item_state = "spickaxe"
-	digspeed = 30 //mines faster than a normal pickaxe, bought from mining vendor
+	digspeed = 20 //mines faster than a normal pickaxe, bought from mining vendor
 	origin_tech = "materials=3;engineering=2"
 	desc = "A silver-plated pickaxe that mines slightly faster than standard-issue."
-
+	force = 17
```
Comment (Supermichael777) [inline code/modules/mining/mine_items.dm]: line spacing

File: code/modules/mining/mine_items.dm [hunk 2]
```diff
@@ -81,18 +81,18 @@
...
 /obj/item/weapon/pickaxe/diamond
 	name = "diamond-tipped pickaxe"
 	icon_state = "dpickaxe"
 	item_state = "dpickaxe"
-	digspeed = 20
+	digspeed = 14
 	origin_tech = "materials=4;engineering=3"
 	desc = "A pickaxe with a diamond pick head. Extremely robust at cracking rock walls and digging up dirt."
-
+	force = 19
```
Comment (Supermichael777) [inline code/modules/mining/mine_items.dm]: line spacing

File: code/modules/mining/mine_items.dm [hunk 3]
```diff
@@ -111,14 +111,14 @@
 /obj/item/weapon/pickaxe/drill/diamonddrill
 	name = "diamond-tipped mining drill"
 	icon_state = "diamonddrill"
-	digspeed = 10
+	digspeed = 7
```
Comment (Supermichael777) [inline code/modules/mining/mine_items.dm]: i think this is fast enough to tunnel away from your foes.

**Comment by erwgd** [comment]
You should improve the force of the drills as well.

**Comment by bgobandit** [comment]
:+1:

**Comment by Cheridan** [comment]
>19 force on a one-handed item that fits on your belt

**Comment by PKPenguin321** [comment]
sounds like cheridan needs to H A R D E N U P

**Comment by TechnoAlchemisto** [comment]
I think that blowing your points on a diamond pick just to bring it back to the station as a weapon is a bit of an edge case, also, it's 1 more point than what the chaplain starts with round start for freeeee with no work required, seems pretty fair to me.

**Comment by octareenroon91** [comment]
Can you cake pickaxes back-wearable, please?

**Comment by TechnoAlchemisto** [comment]
I mean, I guess I could but it would be very rarely used in the back slot.

**Comment by TechnoAlchemisto** [comment]
Also not in this PR.

**Comment by KorPhaeron** [comment]
Jokes on everyone involved they're still gonna be garbage

>3k points for a force 19 weapon

>can buy boxes of force 20 weapons from cargo for probably 1/5 the cost

**Pull request merged**